### ISSUE TITLE
Suggestions to deploy "tomorrow" on Thursday evenings

### ISSUE THREAD

**Issue by nleduc** [baires/shouldideploy#102]
If you load it up on a Thursday evening, it'll still suggest you can wait until tomorrow even if "tomorrow" is Friday. I told my product director he can load shouldideploy.today to find out if we can deploy and he saw this on a Thursday evening: "YOU HAVE FULL DAY AHEAD OF YOU TOMORROW!" and now I'm screwed! This bug has now put Pornhub at risk for the weekend. If we accidentally break something that causes issues on Pornhub over the weekend, the stats show that the world will be at higher rate of crime and unwanted pregnancies. Please patch this ASAP. The human race and the yet-to-be-born will thank you.

**Comment by mikk150** [comment]
that is done so that your superior will be happier, otherwise he says that he is smarter than some random page on web when some random page on web tells that on Thursday evening next deploy day is Monday.

Think about it, if your superior wanted you to deploy change, that notifies users not to have intercourse with entities which has some STD's and it breaks really major notification that Davie504bASS is showing off his "12 inches D" and your superior said that it cannot wait until monday... Welcome back to everyday deployment

TLDR: tell him/her to read it EACH time before deploy(it always
